src/content/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks.mdx

```diff
@@ -142,11 +144,17 @@ New Relic is expanding its support for AI/ML libraries.
 
 | Library | Gem Name(s) | Supported | Notes |
 | ------- | ----------- | --------- | ----- |
-| **GraphQL** | `graphql` |   | Instrumentation provided by GraphQL gem |
-| **Concurrent Ruby** | `concurrent-ruby` | 1.1.5 and above |  |
 | **ActionCable** | `actioncable` | See Rails versions|  |
 | **ActiveMerchant** | `activemerchant`| 1.65.0 and above | ActiveMerchant 1.25.0 - 1.64.x: Last fully supported by agent v8.16.0. |
+| **Capistrano** | `capistrano` | 2.15.5 and above |  |
+| **Concurrent Ruby** | `concurrent-ruby` | 1.1.5 and above |  |
+| **Fiber** |  |  | Instrumentation of Ruby's Fiber class. |
+| **GraphQL** | `graphql` |   | Instrumentation provided by GraphQL gem |
+| **Lambda (AWS SDK)** | `aws-sdk-lambda`|  | See [AWS Lambda Monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/introduction-lambda/) for more information. |
+| **Logger** |  |  | Instrumentation of Ruby's Logger class. |
```

Comment:
Since `logger` is moving out of the standard library, I wonder if we should list its gem name here? 
```suggestion
| **Logger** | `logger` |  | Support for APM log forwarding using Ruby's Logger library. |
```